test/test_quantized.py

```diff
@@ -675,6 +675,7 @@ class TestDynamicQuantizedLinear(TestCase):
         use_channelwise=st.booleans())
     def test_qlinear(self, batch_size, input_channels, output_channels,
                      use_bias, use_relu, use_multi_dim_input, use_channelwise):
+        torch.backends.quantized.engine = torch.fbgemm
```

Comment:
same comment that you'd need a context manager probably

Reply:
I think it'd be even better to lift it into the test's base class to make sure that we run tests agains both engines without duplicating the tests